Sources/FoundationEssentials/AttributedString/FoundationAttributes.swift

```diff
@@ -507,7 +514,21 @@ extension AttributeScopes.FoundationAttributes {
             case nanoseconds
         }
     }
-    
+
+    /// The attribute key for the base writing direction of a paragraph.
+    #if FOUNDATION_FRAMEWORK
+    @_spi(BringUpWritingDirectionAPI)
+    #endif
+    @available(FoundationPreview 6.2, *)
+    @frozen
+    public enum WritingDirectionAttribute: CodableAttributedStringKey {
+        public typealias Value = AttributedString.WritingDirection
+        public static let name: String = "Foundation.WritingDirection"
```

Comment:
I know that we're a bit inconsistent with the existing attributes today, but most attributes use the form `Foundation.WritingDirectionAttribute` as the name rather than `Foundation.WritingDirection`. Could we update this to follow that convention just to try to stick with what the majority of attributes use?